plugins/bot-sentry/bot-sentry.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "bot-sentry",
+  "sources": [
+    {
+      "type": "archive",
+      "url": "https://downloads.sourceforge.net/project/pidgin-bs/bot-sentry/1.3.0/bot-sentry-1.3.0.tar.bz2",
+      "sha256": "e65581328a508a82cd32d246f4037ea3b044eafd223c3c967947f847db28e736"
+    },
+    {
+      "type": "file",
+      "path": "config.guess"
+    },
+    {
+      "type": "file",
+      "path": "config.sub"
+    }
```

Comment:
what we do usually is to copy then from the Sdk.

Like this 
https://github.com/flathub/org.ardour.Ardour/blob/master/org.ardour.Ardour.json#L287